examples/tools/ros/rosenv/workspace/consumer/src/hello.cpp

```diff
@@ -0,0 +1,29 @@
+#include <iostream>
+#include <string>
+
+#include "hello.h"
+
+#include "Box2D/Box2D.h"
+#include <boost/json.hpp>
```

Comment:
As the example is relatively trivial, don't we want to simplify it a bit, and use something less heavy than boost? Specially because there is a ``--build=missing`` that might be building things from source if some config doesn't match our pre-built binaries in ConanCenter.
Both for our CI efficiency, but also for UX of users trying this example, if they have to wait for a long time because boost is being built from source.
I think a simpler example, only with box2d, or fmt or something simple might be a bit better.

Reply:
the only reason to use boost here is to prove that the example can actually consume a library from conan that might be also installed in the system. However, it is not necessary at all to illustrate how the integration works. I will leave just box2d or add something fancy with fmt as suggested 👍